petastorm/tests/test_common.py

```diff
@@ -50,6 +50,7 @@
     UnischemaField('matrix_nullable', np.uint16, _DEFAULT_IMAGE_SIZE, NdarrayCodec(), True),
     UnischemaField('sensor_name', np.unicode_, (1,), NdarrayCodec(), False),
     UnischemaField('string_array_nullable', np.unicode_, (None,), NdarrayCodec(), True),
+    UnischemaField('integer_nullable', np.int32, (), ScalarCodec(ShortType()), True),
```

Comment:
One more reason to consider more "organized" schema where you can set flags on fields :-)